2.3.1/Dockerfile

```diff
@@ -121,8 +121,13 @@ COPY docker-entrypoint.sh /usr/local/bin
 RUN ln -s usr/local/bin/docker-entrypoint.sh /docker-entrypoint.sh # backwards compat
 ENTRYPOINT ["tini", "--", "/docker-entrypoint.sh"]
 
-# Setup directories and permissions
-RUN find /opt/couchdb \! \( -user couchdb -group couchdb \) -exec chown -f couchdb:couchdb '{}' +
```

Comment:
I'm worried that dropping this line this will cause a regression of #131 when starting as root and running as the CouchDB user.

Reply:
I didn't notice any issue with startup times when testing but have added this line back for consistency.

Reply:
I couldn't repro the original issue myself, either. I think it mostly showed up when running with relatively slow PVs underneath.

Reply:
Yeah, specifically those mounted in network file systems (NFS or slow SAN).

Reply:
Ok - I've added this line back at https://github.com/apache/couchdb-docker/pull/151/commits/f9a639120bda80e3b252b089cd50aa33483ed148#diff-ff57d65ca713e117d90559824747793aR127